specification/batch/data-plane/Microsoft.Batch/stable/2022-01-01.15.0/BatchService.json

```diff
@@ -16644,6 +16692,14 @@
           "title": "The reference to the user assigned identity to use to access Azure Blob Storage specified by containerUrl",
           "description": "The identity must have write access to the Azure Blob Storage container",
           "$ref": "#/definitions/ComputeNodeIdentityReference"
+        },
+        "uploadHeaders": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HttpHeader"
+          },
+          "title": "A list of name-value pairs for headers to be used in uploading output files",
+          "description": "These headers will be specified when uploading files to Azure Storage. Official document on allowed headers when uploading blobs: https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob#request-headers-all-blob-types"
```

Comment:
Wording suggestion: "Official document on allowed headers" -> "For more information, see [Request Headers (All Blob Types)](https://docs.microsoft.com/rest/api/storageservices/put-blob#request-headers-all-blob-types)."

Reply:
(This is just following a pattern I see for other swagger files linking to docs)